### PR TITLE
Team G Wiki Entry 2019: GP, GMM

### PR DESCRIPTION
**Pull request by yshi329**
Added a Gaussian Process and Gaussian Mixture Model tutorial

### CODE REVIEW

File: wiki/math/gaussian-process-gaussian-mixture-model.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+7. [Summary](#Summary)
+8. [References](#References)
+
+## Gausssian Process
+
+### What is GP?
+A Gaussian Process(GP) is a probability distribution over functions \\( p(\textbf{f}) \\) , where the functions are defined by a set of random function variables \\(\textbf{f} = \{f_1, f_2, . . . , f_N\}\\). For a GP, any finite linear combination of those function variables has a joint (zero mean) Gaussian distribution. It can be used for nonlinear regression, classification, ranking, preference learning, ordinal regression. In robotics, it can be applied to state estimation, motion planning and in our case environment modeling.
+
+GPs are closely related to other models and can be derived using bayesian kernel machines, linear regression with basis functions, infinite multi-layer perceptron neural networks (it becomes a GP if there are infinitely many hidden units and Gaussian priors on the weights), spline models. A simple connections between different methods can be found in the following figure.
+<span style="display:block;text-align:center">![](https://i.imgur.com/HBIqhsR.png)</span>
```
Comment (smbryan) [inline wiki/math/gaussian-process-gaussian-mixture-model.md]: You're mixing HTML in with the Markdown here and in the commented out line below. Jekyll doesn't handle these well. These should be stripped. 

Also, images should be put in and assets folder and not rely on third-party sources (such as imgur).

**Comment by smbryan** [comment]
This is a strong submission. However, the formatting isn't to our standards. Please double check the [README](https://github.com/RoboticsKnowledgebase/roboticsknowledgebase.github.io/blob/master/README.md) and the [template](https://raw.githubusercontent.com/RoboticsKnowledgebase/roboticsknowledgebase.github.io/master/_templates/template.md) example.

**Comment by yshi329** [comment]
Just updated the format and put images into the asset folder :)

**Comment by smbryan** [comment]
@yshi329, looks good!

**Pull request merged**